src/Sentry.Unity/UnitySentryOptions.cs

```diff
@@ -12,6 +12,18 @@ public enum SentryUnityCompression
         Fastest = 2,
         NoCompression = 3
     }
+
+    // We need to think carefully about what types should be placed here, because of ILL2CPP stripping
+    public sealed class UnitySentryOptionsJson
```

Comment:
Can we drop this type and have only `SentryUnityOptions` though?

Reply:
This is what I tried first, but because of some types in `SentryUnityOptions` itself, `IL2CPP` builds were failing, that's why I included a comment. I need to check which field(-s) exactly messes up `IL2CPP` and what additional lib should be included into `link.xml`. I decided to no waste too much time here, as `IL2CPP` build are **slow**.

Reply:
I see. Yeah IL2CPP is extremely slow. I believe these builds should be a lot faster on the latest Unity version. So perhaps for dev it's best to use something latest. CI we can continue on an older one to make sure we're compatible.